.github/workflows/build.yml

```diff
@@ -44,14 +44,14 @@ jobs:
             arch: 'arm64'
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
> I'm tired of warnings for our workflow runs.

Just asking: what is the source of warnings (gmail letters or some special GitHub panel)? I'd probably need to setup the same warning.

Reply:
Just open something like https://github.com/nspcc-dev/neo-go/actions/runs/7962325628?pr=3319